src/libs/runtime-utils.js

```diff
@@ -69,6 +69,11 @@ export const runtimeCost = ({ runtimeConfig, status }) => {
   }
 }
 
+export const getGalaxyCost = app => {
+  return hourlyKubernetesAppCost(app || { kubernetesRuntimeConfig: { machineType: 'n1-standard-8' } }) +
+    persistentDiskCost({ size: 250 + 10 + 100, status: 'Running' }) // 250Gb for the NFS disk, 10Gb for the postgres disk, and 100Gb for the boot disk
```

Comment:
TOL: Are these numbers available via the persistent disk list that we get back? If so, it might be nice to compute the cost from there at some point (though it would mean wiring in the disks here).

Reply:
I'm not seeing these in the list, though this might be a needed piece in the future. I will make a note, thanks!